Rakefile

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require 'logger'
```

Comment:
I don't think this is required for Rails 7.1+. It was caused by `concurrent-ruby` v3.5 removing the dependency, but was fixed in Rails 7.1 I think and since we're only testing with that version and above I believe we can skip this. (https://github.com/ruby-concurrency/concurrent-ruby/pull/1062)